site-docker.yml.sample

```diff
@@ -19,45 +19,63 @@
   become: True
   gather_facts: false
   roles:
-  - ceph-mon
+    - ceph-defaults
+    - ceph-docker-common
+    - ceph-mon
   serial: 1 # MUST be '1' WHEN DEPLOYING MONITORS ON DOCKER CONTAINERS
 
 - hosts: osds
   become: True
   roles:
-  - ceph-osd
+    - ceph-defaults
+    - ceph-docker-common
+    - ceph-osd
 
 - hosts: mdss
   become: True
   roles:
-  - ceph-mds
+    - ceph-defaults
+    - ceph-docker-common
+    - ceph-mds
 
 - hosts: rgws
   become: True
   roles:
-  - ceph-rgw
+    - ceph-defaults
+    - ceph-docker-common
+    - ceph-rgw
 
 - hosts: nfss
   become: True
   roles:
-  - ceph-nfs
+    - ceph-defaults
+    - ceph-docker-common
+    - ceph-nfs
 
 - hosts: rbd_mirrors
   become: True
   roles:
-  - ceph-rbd-mirror
+    - ceph-defaults
+    - ceph-docker-common
+    - ceph-rbd-mirror
 
 - hosts: restapis
   become: True
   roles:
-  - ceph-restapi
+    - ceph-defaults
+    - ceph-docker-common
+    - ceph-restapi
 
 - hosts: mgrs
   become: True
   roles:
+    - ceph-defaults
```

Comment:
``ceph-defaults`` and ``ceph-docker-common`` should only be used ``when: "ceph_release_num.{{ ceph_stable_release }} > ceph_release_num.jewel"``